platform-emf/src/main/java/com/softicar/platform/emf/form/section/EmfFormSectionDiv.java

```diff
@@ -93,18 +91,11 @@ public void setInvisibleMode() {
 	public void display(Exception exception) {
```

Comment:
extracted class `EmfFormExceptionDiv` and added special handling of `EmfOutdatedTableRowException`